src/plugins/intel_gna/tests/unit/ops/util_test.cpp

```diff
@@ -0,0 +1,156 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstdint>
+#include <vector>
+#include <thread>
+#include <memory>
+
+#include <gtest/gtest.h>
+#include "ops/util/util.hpp"
+
+using namespace ov::intel_gna::ngraph_util;
+
+namespace test {
+
+using GnaOpsUtilTestParams = std::tuple<std::shared_ptr<ngraph::Node>, bool>;
+
+class GnaOpsUtilTest : public ::testing::TestWithParam<GnaOpsUtilTestParams> {
+public:
+    static std::string get_test_name(const testing::TestParamInfo<GnaOpsUtilTestParams>& obj) {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = obj.param;
+
+        std::ostringstream test_name;
+        test_name << "Node=" << node->get_type_info().name << "_";
+        test_name << "Op=" << node->get_type_info().get_version() << "_";
+        test_name << "ExpectedReulst=" << result;
+
+        return test_name.str();
+    }
+};
+
+class GnaOpsUtilisPoolingTest : public GnaOpsUtilTest {
+public:
+    void validate() {
+        std::shared_ptr<ngraph::Node> node;  // node to be converted
+        bool result; // expected result
+        std::tie(node, result) = GetParam();
+        ASSERT_TRUE(result == is_pooling(node));
```

Comment:
There is a macro EXPECT_EQ(). Maybe it is better than general ASSERT_TRUE?

Reply:
Maybe, but what will be the difference?